ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Tests/Timeouts.hs

```diff
@@ -58,24 +62,16 @@ prop_timeouts = do
 
   where
 
-    scSlotLength :: SlotLength
-    scSlotLength = slotLengthFromSec 20
-
     -- A schedule that advertises all the points of the chain from the start but
     -- contains just one too many ticks, therefore reaching the timeouts.
-    dullSchedule :: DiffTime -> TestFrag -> PointSchedule
-    dullSchedule _ (AF.Empty _) = error "requires a non-empty block tree"
-    dullSchedule timeout (_ AF.:> tipBlock) =
+    dullSchedule :: PointScheduleConfig -> DiffTime -> TestFrag -> PointSchedule
+    dullSchedule _ _ (AF.Empty _) = error "requires a non-empty block tree"
+    dullSchedule scheduleConfig timeout (_ AF.:> tipBlock) =
       let tipPoint = TipPoint $ tipFromHeader tipBlock
           headerPoint = HeaderPoint $ getHeader tipBlock
           blockPoint = BlockPoint tipBlock
           state = Peer HonestPeer $ NodeOnline $ AdvertisedPoints tipPoint headerPoint blockPoint
           tick = Tick { active = state, peers = Peers state Map.empty }
-          maximumNumberOfTicks = fromIntegral $ roundDiffTimeToSeconds $ timeout / pscTickDuration scSchedule
+          maximumNumberOfTicks = fromIntegral $ round $ timeout / pscTickDuration scheduleConfig
```

Comment:
I think `fromIntegral` is redundant since `round` can return `Int`.

Reply:
It was removed later on.